src/TranscriptRow/stories/index.stories.js

```diff
@@ -35,8 +37,8 @@ storiesOf('Transcript Row', module)
           { ...transcriptItems[0] }
           handleEditItem={ transcriptCardActions.handleEdit }
           handleDeleteItem={ transcriptCardActions.handleDelete }
-          mediaDuration={ 10000 }
-          created={ '20.04.21, 12:53' }
```

Comment:
I think we want to keep the [date], [time] format, this is in the Firebase master: https://github.com/bbc/digital-paper-edit-firebase/blob/master/src/Util/time/index.js

Reply:
Oh thanks for this! I missed the time stamp on the designs as well, my bad